<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1705</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des examens</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-40</y>
      <width>1661</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* === Background with Linear Gradient === */
QWidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, 
                                stop:0 #007BFF, stop:1 #F8F9FA);
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 12pt;
}

/* === Group Box Styling === */
QGroupBox {
    border: 2px solid #0056b3;
    border-radius: 10px;
    margin-top: 10px;
    padding: 15px;
    background: transparent; /* Semi-transparent */
}

/* === Stylish Input Fields === */
QLineEdit, QComboBox, QDateEdit {
    border: 2px solid #007BFF;
    border-radius: 6px;
    padding: 6px;
    background: white;
    transition: all 0.3s ease-in-out;
}

/* === Input Fields on Focus === */
QLineEdit:focus, QComboBox:focus, QDateEdit:focus {
    border: 2px solid #0056b3;
    background: #e6f2ff;
}

/* === Modern Buttons === */
QPushButton {
    background-color: #007BFF;
    color: white;
    border-radius: 10px;
    padding: 10px 20px;
    font-weight: bold;
    border: none;
    transition: all 0.3s ease-in-out;
}

/* === Button Hover Effect === */
QPushButton:hover {
    background-color: #0056b3;
    transform: scale(1.05);
}

/* === Button Click Effect === */
QPushButton:pressed {
    background-color: #003d82;
    transform: scale(0.95);
}

/* === Table Styling === */
QTableWidget {
    background: white;
    border: 2px solid #007BFF;
    gridline-color: #007BFF;
    selection-background-color: #CDE1F8;
}

/* === Headers in Table === */
QHeaderView::section {
    background-color: #007BFF;
    color: white;
    font-weight: bold;
    padding: 5px;
    border: 1px solid #0056b3;
}

/* === Scrollbar for Table === */
QScrollBar:vertical, QScrollBar:horizontal {
    background: white;
    border: 2px solid #007BFF;
    width: 12px;
    border-radius: 6px;
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #007BFF;
    border-radius: 6px;
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:horizontal:hover {
    background: #0056b3;
}

QLabel{
background-color:transparent;
}

QFormLayout{
background-color:transparent;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="ajout_examButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>470</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Users/Pc/Desktop/Downloads/projetcpp-2a33-smart-pharmacy-gestioncommande/projetcpp-2a33-smart-pharmacy-gestioncommande/461755239_1311673180207872_9007432977068590092_n.png</normaloff>../Users/Pc/Desktop/Downloads/projetcpp-2a33-smart-pharmacy-gestioncommande/projetcpp-2a33-smart-pharmacy-gestioncommande/461755239_1311673180207872_9007432977068590092_n.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>1340</x>
        <y>490</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Exporter PDF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Saisir le titre</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_examens">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>130</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #007BFF;</string>
      </property>
      <column>
       <property name="text">
        <string>Titre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sujet</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Statut</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher par titre:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="suppexamButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>470</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../Users/Pc/Desktop/Downloads/460177168_1665954550921902_7047727999174566408_n.png</normalon>
       </iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="modifexamButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../Users/Pc/Desktop/Downloads/461508595_825905609429817_3860263783025879477_n.png</normalon>
       </iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Titre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>48</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Sujet:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Statut:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_titre">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>311</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sujet">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>311</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_statut">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>311</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>1320</x>
        <y>130</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxsortchamp">
      <property name="geometry">
       <rect>
        <x>1300</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <item>
       <property name="text">
        <string>Sujet</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idExam">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>311</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_order">
      <property name="geometry">
       <rect>
        <x>1300</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <item>
       <property name="text">
        <string>Ascendant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Descendant</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="trier_Butoon">
      <property name="geometry">
       <rect>
        <x>1300</x>
        <y>310</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Date </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_transport">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="qrCodeLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>570</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../Downloads/event_planner_2A35-abdelmonam/projetevent/image/icons8-qr-code-64.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>390</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>transport:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_exam">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>480</y>
        <width>681</width>
        <height>271</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1705</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
